documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc

```diff
@@ -158,76 +237,38 @@ spec:
-        annotations:
-          kubeletconfig.experimental: |
-            {"topologyManagerScope": "pod",
-             "systemReserved": {"memory": "3Gi"}
-            }
-      spec:
-        cpu:
-          isolated: '{{hub fromConfigMap "" "hub1-sites-data" (printf "%s-isolated-cpus" .ManagedClusterName) hub}}'
-          reserved: '{{hub fromConfigMap "" "hub1-sites-data" (printf "%s-reserved-cpus" .ManagedClusterName) hub}}'
-        hugepages:
-          defaultHugepagesSize: 1G
-          pages:
-          - count: '{{hub fromConfigMap "" "hub1-sites-data" (printf "%s-hugepages-count" .ManagedClusterName)|toInt hub}}'
-            size: '{{hub fromConfigMap "" "hub1-sites-data" (printf "%s-hugepages-size" .ManagedClusterName) hub}}'
-        numa:
-          topologyPolicy: single-numa-node
-        realTimeKernel:
-          enabled: false
-        globallyDisableIrqLoadBalancing: false
-        # WorkloadHints defines the set of upper level flags for different type of workloads.
-        workloadHints:
-          realTime: true
-          highPowerConsumption: false
-          perPodPowerManagement: false
-    - fileName: TunedPerformancePatch.yaml
-      policyName: "site-policies"
-      spec:
-        profile:
-          - name: performance-patch
-            data: |
-              [main]
-              summary=Configuration changes profile inherited from performance created tuned
-              include=openshift-node-performance-openshift-node-performance-profile
-              [sysctl]
-              kernel.timer_migration=1
-              [scheduler]
-              group.ice-ptp=0:f:10:*:ice-ptp.*
-              group.ice-gnss=0:f:10:*:ice-gnss.*
-              [service]
-              service.stalld=start,enable
-              service.chronyd=stop,disable
+              deviceSelector:
+                paths:
+                - '{{hub fromConfigMap "" "site-data-configmap" (printf "%s-storage-path" .ManagedClusterName) hub}}'
+      policyName: "sites-policy"
 EOF
 -----
 +
-2. We're using policy templating, so we need to create the `ConfigMap` with the templating values to be used:
+2. We're using policy templating, so we need to create the `ConfigMap` with the templating values to be used. Observe that the following manifest contains disk paths for clusters with the same hardware specification: hw-type-platform-1. In the policy we have just applied we are capturing the device path to be used for persistent storage by obtaining the name of the cluster managed by the hub. This is not a label set in the `SiteConfig` definition, it is the `clusterName` value instead. Check the link:crafting-deployments-iaac.html#siteconfig[previous section].
+
+IMPORTANT: Leveraging hub site templating we are reducing the number of policies on our hub cluster. This makes the clusters's deployment and maintenance process a lot less cumbersome. Notice that with large fleets of clusters, it can quickly become hard to maintain per-site configurations.
```

Comment:
```suggestion
IMPORTANT: By leveraging hub site templating we are reducing the number of policies on our hub cluster. This makes the clusters's deployment and maintenance process a lot less cumbersome. Notice that with large fleets of clusters, it can quickly become hard to maintain per-site configurations.
```